common/smoke-tests/smoke-test-dependencies.yml

```diff
@@ -70,30 +70,30 @@ jobs:
           PomFile: common/smoke-tests/pom.xml
           ServiceDirectory: '$(Build.SourcesDirectory)/common/smoke-tests/'
         Java 8 Windows (AzureUSGovernment):
-          Pool: "azsdk-pool-mms-win-2019-general"
+          Pool: "azsdk-pool-mms-win-2022-general"
           OSVmImage: "MMS2019"
           SubscriptionConfiguration: $(sub-config-gov-test-resources)
           ArmTemplateParameters: $(azureUSGovernmentArmParameters)
           PomFile: common/smoke-tests/pom.xml
           ServiceDirectory: '$(Build.SourcesDirectory)/common/smoke-tests/'
         Java LTS Windows (AzureUSGovernment):
-          Pool: "azsdk-pool-mms-win-2019-general"
+          Pool: "azsdk-pool-mms-win-2022-general"
           OSVmImage: "MMS2019"
           SubscriptionConfiguration: $(sub-config-gov-test-resources)
           ArmTemplateParameters: $(azureUSGovernmentArmParameters)
           PomFile: common/smoke-tests/pom.xml
           ServiceDirectory: '$(Build.SourcesDirectory)/common/smoke-tests/'
         Java 8 Windows (AzureChinaCloud):
-          Pool: "azsdk-pool-mms-win-2019-general"
+          Pool: "azsdk-pool-mms-win-2022-general"
           OSVmImage: "MMS2019"
           SubscriptionConfiguration: $(sub-config-cn-test-resources)
           ArmTemplateParameters: $(azureChinaCloudArmParameters)
           PomFile: common/smoke-tests/pom.xml
           ServiceDirectory: '$(Build.SourcesDirectory)/common/smoke-tests/'
         Java LTS Windows (AzureChinaCloud):
-          Pool: "azsdk-pool-mms-win-2019-general"
+          Pool: "azsdk-pool-mms-win-2022-general"
           OSVmImage: "MMS2019"
           SubscriptionConfiguration: $(sub-config-cn-test-resources)
           ArmTemplateParameters: $(azureChinaCloudArmParameters)
           PomFile: common/smoke-tests/pom.xml
-          ServiceDirectory: '$(Build.SourcesDirectory)/common/smoke-tests/'
+          ServiceDirectory: '$(Build.SourcesDirectory)/common/smoke-tests/'
```

Comment:
@sima-zhu did you not figure out how to leave the line-ending at the end of file?